common/flatpak-context.c

```diff
@@ -2435,10 +2435,65 @@ flatpak_context_make_sandboxed (FlatpakContext *context)
 }
 
 const char *dont_mount_in_root[] = {
-  ".", "..", "lib", "lib32", "lib64", "bin", "sbin", "usr", "boot", "efi",
-  "root", "tmp", "etc", "app", "run", "proc", "sys", "dev", "var", NULL
+  ".",
+  "..",
+  "app",
+  "bin",
+  "boot",
+  "dev",
+  "efi",
+  "etc",
+  "lib",
+  "lib32",
+  "lib64",
+  "proc",
+  "root",
+  "run",
+  "sbin",
+  "sys",
+  "tmp",
+  "usr",
+  "var",
+  NULL
 };
 
+static void
+cannot_export (FlatpakFilesystemMode  mode,
```

Comment:
Tiny bikeshed. I'd find this function more clear if it noted that its just a logging function.

`print_cannot_export_error`, `log_cannot_export_error`, etc

Reply:
Good thinking, I used `log_cannot_export_error`.